### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by agnarrarendelle**
Hi, I have fixed a few typos in commands. Please review my pr.

### CODE REVIEW

File: compiler/rustc_middle/src/ty/consts/valtree.rs [hunk 1]
```diff
@@ -23,8 +23,8 @@ pub enum ValTree<'tcx> {
     /// of these types have the same representation.
     Leaf(ScalarInt),
 
-    //SliceOrStr(ValSlice<'tcx>),
-    // dont use SliceOrStr for now
```
Comment (WaffleLapkin) [inline compiler/rustc_middle/src/ty/consts/valtree.rs:27]: this is not a typo, this is commented out code.

**Review by WaffleLapkin — CHANGES_REQUESTED** [review]

File: compiler/rustc_middle/src/ty/consts/valtree.rs (hunk 1 above)
Comment (agnarrarendelle) [reply compiler/rustc_middle/src/ty/consts/valtree.rs:27]: Sorry, fixed it now

Comment (WaffleLapkin) [reply compiler/rustc_middle/src/ty/consts/valtree.rs:27]: still not fixed, `// dont use SliceOrStr for now` is also part of the comment and not documentation

Comment (agnarrarendelle) [reply compiler/rustc_middle/src/ty/consts/valtree.rs]: I have undone everything besides changing `// dont ` to `// don't`. Please have a look.

**Review by WaffleLapkin — APPROVED** [review]
Looks fine to me. Please squash commits together.

**Comment by agnarrarendelle** [comment]
I have combined the commits together now

**Comment by workingjubilee** [comment]
Thanks!
@bors r+ rollup

**Pull request merged**